host/xtest/stats.c

```diff
@@ -312,6 +313,36 @@ static int stat_loaded_ta(int argc, char *argv[])
 	return close_sess(&ctx, &sess);
 }
 
+static int stat_system_time(int argc, char *argv[])
+{
+	TEEC_Context ctx = { };
+	TEEC_Session sess = { };
+	TEEC_Result res = TEEC_ERROR_GENERIC;
+	uint32_t eo = 0;
+	TEEC_Operation op = { };
+
+	UNUSED(argv);
+	if (argc != 1)
+		return usage();
+
+	open_sess(&ctx, &sess);
+	op.paramTypes = TEEC_PARAM_TYPES(TEEC_VALUE_OUTPUT,
+					 TEEC_VALUE_OUTPUT,
+					 TEEC_NONE, TEEC_NONE);
+	res = TEEC_InvokeCommand(&sess, STATS_CMD_SYSTEM_TIME, &op, &eo);
```

Comment:
This command could be implemented in the "OS test" TA (`ta/os_test/os_test.c`).

Reply:
> This command could be implemented in the "OS test" TA (`ta/os_test/os_test.c`).

Thanks for your comment. It seems "OS test" TA hasn't been included into our really product. Can we keep it in ``stats`` PTA?

Reply:
Please ignore my suggestion, we'll keep the thing in the stats PTA.